ui/component/commentReactions/index.js

```diff
@@ -15,6 +16,7 @@ const select = (state, props) => {
 
   return {
     claim,
+    disableReactions: makeSelectTagInClaimOrChannelForUri(props.authorUri, DISABLE_REACTIONS_ALL_TAG)(state),
```

Comment:
what happens if authoruri is blank? Anon claim. Does something handle that downstream?

Reply:
Seems so, didn't looked the code, but trying `undefined` there works just fine. Ends up returning something falsy, which is wanted.